content/en/blog/_posts/xxxx-xx-xx-Fine-grained-SupplementalGroups-control/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: blog
+title: 'Kubernetes 1.31: Fine-grained SupplementalGroups control'
+date: xxxx-xx-xx
```

Comment:
Insert the 22nd of August here (and in the folder name)

Reply:
fixed in https://github.com/kubernetes/website/pull/46921/commits/14612bb32776a7b740db3adddfa8600f6a6760f4